pom.xml

```diff
@@ -159,6 +159,12 @@
 
     <dependencies>
 
+        <dependency>
+            <groupId>org.roboy.memory</groupId>
+            <artifactId>roboy_memory</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+
```

Comment:
When you run mvn clean install, it is automatically added to the local maven repo @`~/.m2`. Long term, it might be worth considering pushing it into a Roboy Maven server or the Maven Central (see SDE 42), but currently it will be used as a dependency